### ISSUE TITLE
unnecessary messages reported by ktest

### ISSUE THREAD

**Issue by grosu** [kframework/k-legacy#375]
Why does ktest report the second and third messages below?

```
D:\github.grosu\k\tutorial\1_k\4_imp++\lesson_4>ktest tests\config.xml --skip=pdf
Kompile the language definitions...(1 in total)
SUCCESS
Kompiling definitions that are not yet kompiled.
Kompile the language definitions...(0 in total)
SUCCESS
```

When one kompiles only one definition, there is no need for those messages. They just make reading the output harder.

**Comment by osa1** [comment]
@grosu with the recent change can you try again and close this if you're happy with the result?

**Comment by grosu** [comment]
Sorry, but I don't know how to test this.  Your changes, Omer, so not seem to have been applied to the unified-builtins2 branch, which my pull request was following.  I tried to pull from kframework/k in my branch and I managed to run into several conflicts.  Sorry, but I don't know how to deal with this.  Hopefully somebody can merge the current master into the unified-builtins2 branch, so I can pull into my branch?

Thanks,
Grigore
